csharp/src/Google.Protobuf.Test/AsyncOnlyStreamWrapper.cs

```diff
@@ -0,0 +1,48 @@
+﻿#if !PROTOBUF_NO_ASYNC
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Google.Protobuf
```

Comment:
Was going to suggest we moved it to Google.Protobuf.Test, but apparently everything's in Google.Protobuf. Apologies for that :(